### PR TITLE
feat(k8s-libs): add version 1.28 and 1.29

### PR DESCRIPTION
**Pull request by efbar**
Ref: https://github.com/influxdata/sre/issues/3221

It adds k8s v1.28 and v1.29 jsonnet libs.
It downgrade `jsonnet-libs-k8s` docker image because v0.0.7 doesn't support arm

### CODE REVIEW

**Comment by efbar** [comment]
@emoruzzi i had to downgrade the `quay.io/influxdb/jsonnet-libs-k8s` image due to:
```
λ make all
docker pull quay.io/influxdb/jsonnet-libs-k8s:0.0.7-rc0
0.0.7-rc0: Pulling from influxdb/jsonnet-libs-k8s
no matching manifest for linux/arm64/v8 in the manifest list entries
make: *** [build] Error 1
```

**Comment by emoruzzi** [comment]
@efbar this will build the container with `linux/amd64,linux/arm64`, feel free to approve and merge. Once merged, it'll create 0.0.9.

https://github.com/influxdata/k8s-infra-containers-build/pull/165

**Review by logan-bobo — APPROVED** [review]

File: Makefile [hunk 1]
```diff
@@ -1,6 +1,6 @@
 IMAGE_NAME ?= jsonnet-libs-k8s
 IMAGE_PREFIX ?= quay.io/influxdb
-IMAGE_TAG ?= 0.0.7-rc0
+IMAGE_TAG ?= 0.0.8-rc0
```
Comment (efbar) [inline Makefile:3]: @emoruzzi now it is at the right version (it works with M1)

File: Makefile [hunk 2]
```diff
@@ -1,6 +1,6 @@
 IMAGE_NAME ?= jsonnet-libs-k8s
 IMAGE_PREFIX ?= quay.io/influxdb
-IMAGE_TAG ?= 0.0.7-rc0
+IMAGE_TAG ?= 0.0.9-rc0
```
Comment (efbar) [inline Makefile:3]: now it is really at the right version

**Review by emoruzzi — APPROVED** [review]

**Pull request merged**